eZ/Publish/Core/Search/Common/FieldValueMapper.php

```diff
@@ -27,11 +27,11 @@
     abstract public function canMap(Field $field);
 
     /**
-     * Map field value to a proper Elasticsearch representation.
+     * Map field value to a proper Solr representation.
```

Comment:
Oops, missed quite a bit of those, thanks. Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/1697/commits/fd99df72bdd5f7cc2a71f74527e869836905b4f4